sdk/search/azure-search-documents/tests/async_tests/test_service_live_async.py

```diff
@@ -57,6 +60,8 @@ async def test_get_service_statistics(self, api_key, endpoint, **kwargs):
         assert isinstance(result, dict)
         assert set(result.keys()) == {"counters", "limits"}
 
+    # Index operations
```

Comment:
How about putting different tests of operations into different files with the operation in the file names?

Then customers can easily know what the tests are testing about.

Reply:
That seems reasonable, I'd like to do it in a different PR